assets/src/console/pages/gateway-accounts/CardTypesPage.tsx

```diff
@@ -97,6 +100,7 @@ const CardTypesPage: React.FC<Props> = ({ service, gatewayAccount }) => {
               cardTypeIds: values.cardTypes
             }
           });
+          setInitialCardValues(values.cardTypes);
```

Comment:
have apollo update the cache instead of storing this separately

https://www.apollographql.com/docs/react/data/mutations/#updating-the-cache-after-a-mutation

Reply:
Apollo cache solution worked out. But need to fix the flickering issue on the front end while doing submit.